capstone/src/main/java/com/google/step/servlets/RestaurantServlet.java

```diff
@@ -0,0 +1,137 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.servlets;
+
+import com.google.appengine.api.datastore.GeoPt;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.step.clients.RestaurantClient;
+import com.google.step.data.Restaurant;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet responsible for sending/getting a restaurant to/from Datastore. */
```

Comment:
Nit: Unless readers need to know that the backing datastore is Datastore, it's preferred to avoid mentioning it. This way if you change the backing data store, you don't have to go update lots of comments.

Reply:
Oh, sorry. This isn't my change. I think I'm using git really wrong, but I'm not really sure what I'm doing wrong. I never actually touched `RestaurantServlet.java` until [PR #24 ](https://github.com/googleinterns/step93-2020/pull/24).That's been there since the time when the servlet actually had Datastore as a part of it. I'll fix this in the other pull request where I actually alter `RestaurantServlet.java`.